airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_14_0.java

```diff
@@ -76,7 +76,6 @@ public void migrate(Map<ResourceId, Stream<JsonNode>> inputData, Map<ResourceId,
       final Consumer<JsonNode> recordConsumer = outputData.get(entry.getKey());
       entry.getValue().forEach(recordConsumer);
     }
-    MigrationUtils.registerMigrationRecord(outputData, JobKeys.AIRBYTE_METADATA.toString(), getVersion());
```

Comment:
example of how we were haphazardly handling this metadata. you'll see it removed in 3 other places in this PR. even then this wasn't handling updating the airyte version, it was just injecting the migration metadata.